### DISCUSSION TITLE
Removing 'with' statements

### DISCUSSION THREAD

**Discussion by fhetextbook** [babel/babel#13074] [General] [answered]
Hi,

I wonder if there is a standard Babel plugin that can remove `with` statements. I found ([babel-plugin-transform-with](https://github.com/pugjs/babel-plugin-transform-with)), but this one wraps variables with an anonymous function, which is problematic in my application.

**Comment by nicolo-ribaudo** [comment, marked as answer]
No: because standard Babel plugins only compile ES5+ features, but `with` statements have been supported since the first JavaScript version (in 1997).
